players.md

```diff
@@ -0,0 +1,85 @@
+# Player Differences
+
+Swiftfin offers two player options: the default Swiftfin player and Native (AVKit). The Swiftfin team recommends using the Swiftfin player for optimal compatibility and features, though the Native player is also available for certain cases that benefit from Apple's native capabilities. All video, audio, and subtitle formats listed are supported for direct playback. If transcoding is enabled on your server, any unsupported formats will be converted automatically.
+
+---
+
+## Feature Support
+
+| Feature                 | Swiftfin (VLCKit)                                                                                               | Native (AVKit)                                                               |
+|-------------------------|-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|
+| **Framerate Matching**  | ❌                                                                                                             | ✅                                                                           |
+| **HDR Playback**        | ❌                                                                                                             | ✅ **                                                                        |
```

Comment:
Might be good to call out HDR supported formats: HDR10, HDR10+ (on certain devices like Apple TV 4k 3rd gen), Dolby Vision profiles 5, 8.
As a note for future, unsure about HLG and Dolby Vision profile 10 (used in AV1)